scripts/Muon/GUI/Common/ADSHandler/ADS_calls.py

```diff
@@ -26,3 +27,11 @@ def check_if_workspace_exist(name):
 
 def add_ws_to_ads(name, workspace):
     AnalysisDataService.addOrReplace(name, workspace)
+
+
```

Comment:
Is this one of those fun Github errors and there isn't actually two empty lines here?

Reply:
there are 2 empty lines for pep8 (2 lines between functions)